frontend/src/components/LoginNavBar.tsx

```diff
@@ -10,26 +10,37 @@ import css from './LoginNavBar.module.scss';
 const classes = classNames.bind(css);
 
 export const LoginNavBar: React.FC<{
-  type: 'landing' | 'login' | 'register';
+  type: 'landing' | 'login' | 'register' | 'getStarted';
 }> = ({ type }) => (
   <div
     className={classes(css.loginNavBar, {
       [css.landingNavBar]: type === 'landing',
     })}
   >
-    {type === 'landing' ? (
+    {type === 'landing' || type === 'getStarted' ? (
       <VisdomLogo />
     ) : (
       <Link to={paths.home} className={classes(css.visdomLogo)}>
         <VisdomLogo />
       </Link>
     )}
     <span />
-    {type === 'login' ? (
+    {type === 'getStarted' && (
+      <Link className={classes(css.loginButton)} to={paths.logoutPage}>
+        <Trans i18nKey="Logout" />
+      </Link>
+    )}
+    {type === 'login' && (
       <Link className={classes(css.loginButton)} to={paths.registerPage}>
         <Trans i18nKey="Register" />
       </Link>
-    ) : (
+    )}
+    {type === 'landing' && (
+      <Link className={classes(css.loginButton)} to={paths.loginPage}>
+        <Trans i18nKey="Login" />
+      </Link>
+    )}
+    {type === 'register' && (
```

Comment:
These could maybe be combined
```suggestion
    {(type === 'register' || type === 'landing') && (
```